Yoast/wordpress-seo
# Decrease the number of cases that can cause changing overall scores

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* This code fixes one situation that can cause changing overall scores. We now trigger a `initializationDone` at the end of the initialization process, which signals to `refreshAnalysis` that the initialization is finished and the analysis can start. Before, the first analysis run was done before the metatitle was available. As a result, the score would change at the moment the title became available and the analysis was reran. However, the previously used keyword assessment is still added later (but see https://github.com/Yoast/wordpress-seo/issues/10741), and the same goes for assessments from external plugins (like Local SEO). 
* I've only implemented this for the post scraper, because the analysis is currently not working for the terms (because of a partial webworker implementation).
* I've removed the claim that the changing of bullet colors is fixed from the changelog item about the loader spinners.

## Test instructions for acceptance testers

This PR can be tested by following these steps:
* console.log the scores in refreshAnalysis.js
<img width="735" alt="schermafbeelding_2018-08-22_om_14_52_38" src="https://user-images.githubusercontent.com/17744553/44464489-3b0f8200-a61b-11e8-9b72-a8ab9b5f7e7a.png">

* Open a post. See #10721 for an example text.  Only 1 score should be logged in the console. It might appear multiple times, but it should be the same number over and over. 
* NB. When you change the focus keyword, the previously used keyword assessment is triggered, which will change the logged score.

## Test instructions for RC testers
* Make sure you only have wordpress-seo active (no add-ons or other plugins that add assessments to our analysis).
* Open an existing post. See #10721 for an example text. Don't touch the keyword input field! (See `Relevant technical choices` )
* The bullets shouldn't change color on page load.

## Quality assurance


Fixes #10721
